src/modules/shared/ProductSlider/slick-theme.scss

```diff
@@ -0,0 +1,132 @@
+@charset "UTF-8";
+@import '../../../styles/blocks/grid';
+@import '../../../styles/utils/variables/colors';
+
+/* Arrows */
+// @include onMobile {
+//   .slick-prev,
+//   .slick-next,
+//   .slick-arrow {
+//     display: none;
+//     visibility: hidden;
+//   }
+// }
+
+// @include onTablet {
+//   .slick-arrow {
+//     position: absolute;
+//     height: 400px;
+//     width: 32px;
+//     cursor: pointer;
+//     border: 1px solid $color__secondary;
+//     border-radius: 48px;
+//     background-color: $color__white;
+//     top: 46.5% ;
+//     -webkit-transform: translate(0, -50%);
+//     -ms-transform: translate(0, -50%);
+//     transform: translate(0, -50%);
+//     padding: 0;
+
+//       &:hover {
+//           border-color: $color__primary;
+//       }
+//   }
+
+//   .slick-prev {
+//     left: -49px;
+//   }
+
+//   .slick-next {
+//       right: -49px;
+//   }
+// }
+
+  // @include onDesktop {
+  //   .slick-arrow {
+  //     position: absolute;
+  //     height: 400px;
+  //     width: 32px;
+  //     cursor: pointer;
+  //     border: 1px solid $color__secondary;
+  //     border-radius: 48px;
+  //     background-color: $color__white;
+  //     top: 46.5% ;
+  //     -webkit-transform: translate(0, -50%);
+  //     -ms-transform: translate(0, -50%);
+  //     transform: translate(0, -50%);
+  //     padding: 0;
+
+  //       &:hover {
+  //           border-color: $color__primary;
+  //       }
+  //   }
+
+  //   .slick-prev {
+  //     left: -49px;
+  //   }
+
+  //   .slick-next {
+  //       right: -49px;
+  //   }
+  // }
+
+/* Dots */
+
+// .slick-dots-product {
+//   display: none;
+//   visibility: hidden;
+// }
+
+// .slick-dots {
+//   margin-top: 8px;
+//   list-style: none;
+//   text-align: center;
+//   width: 100%;
+
+//     li {
+//       position: relative;
+//       display: inline-block;
+
+//       button {
+//         border: 0;
+//         background: transparent;
+//         width: 24px;
+//         height: 24px;
+//         margin: 0 4px;
+//         padding: 0;
+//         cursor: pointer;
+//         color: transparent;
+
+//         &:before {
+//           display: block;
+//           position: absolute;
+//           content: '';
+//           width: 14px;
+//           height: 4px;
+//           margin-top: 7px;
+//           margin-left: 4px;
+//           background-color: $color__elements;
+//           opacity: 1;
+//         }
+//       }
+
+//       &.slick-active button:before {
+//         background-color: $color__primary;
+//         opacity: 1;
+//       }
+//     }
+
+//   &__dark {
+//     li {
+//       button {
+//         &:before {
+//             background-color: #3b3e4a;
+//         }
+//       }
+
+//       &.slick-active button:before {
+//         background-color: #f1f2f9;
+//       }
+//     }
+//   }
+// }
```

Comment:
Why is this commented on?

Reply:
> Why is this commented on?

sorry, I just forgot to delete this file 😅 I'll fix it in the next pr :)